src/client/src/ui/analytics/views/analyticsView.jsx

```diff
@@ -123,7 +120,7 @@ export default class AnalyticsView extends ViewBase {
             onClick={() => router.publishEvent(this.props.modelId, 'togglePnlDisplayMode', {})}>Positions
           </button>
         </div>
-        <div className='nv-container clearfix pnlchart'>
+        <div className='analytics__nv-container clearfix pnlchart'>
```

Comment:
maybe we could use something more meaningfull than nv like analytics__chart-container?